lambdas/user_access_patterns/lambda_for_createUser/index.js

```diff
@@ -1,6 +1,58 @@
-const AWS = require('aws-sdk');
-exports.handler = async (event) => {
-    // TODO implement
-    const response = (success, reason, data) => { return { success: success, reason: reason, data: data } }
-    return response(true, null, null);
+const AWS = require("aws-sdk");
+const crypto = require("crypto");
+//AWS.config.loadFromPath("../../../keys.json");
+//Use the below code for local setup
+AWS.config.update({
+  region: "local",
+  endpoint: "http://localhost:8000",
+});
+
+const dynamo = new AWS.DynamoDB.DocumentClient();
+function response(statusCode, message) {
+  return {
+    statusCode: statusCode,
+    body: JSON.stringify(message),
+  };
+}
+
+function hmac_hash(secretKey, password) {
+  const hashingSecret = secretKey;
+  const plainText = password;
+  const hashedStr = crypto
+    .createHmac("sha256", hashingSecret)
+    .update(plainText)
+    .digest("hex");
+
+  return hashedStr;
+}
+
+module.exports.createUser = (event, ctx, callback) => {
+  const reqBody = JSON.parse(event.body);
+  const user = {
+    u_name: reqBody.u_name,
+    u_age: reqBody.u_age,
+    u_thumbnail: reqBody.u_thumbnail,
+    u_is_verified: true,
+    u_provider: reqBody.u_provider,
+    u_password: hmac_hash(reqBody.u_email, reqBody.u_password),
+    u_space_used: 0,
+    u_user_type: "default",
+  };
+
+  return dynamo
+    .put({
+      TableName: "V-Transfer",
+      Key:{
+        PK:`USER#${reqBody.u_email}`,
```

Comment:
avoid instantiating the keys in place rather use a separate variable.

Reply:
Does it makes a difference?

Reply:
yes, its easy to maintain

Reply:
So you want to have a variable in the same file and then use it for values of PK and SK?

Reply:
yes

Reply:
if you are the owner of PR then let the reviewer decide weather to the conversation is resolved or not, else their would be no point.